src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriorsIntegrationTest.java

```diff
@@ -1,139 +1,90 @@
-        executeTest("testMissingPriors", spec);
+                        " -V " + largeDir + "1000G.phase3.broad.withGenotypes.chr20.10100000.vcf",
+                Collections.singletonList(dir + "expectedCGP_testMissingPriors.vcf")
+        );
+        spec.executeTest("testMissingPriors", this);
     }
 
-    @Test(enabled = true)
-    public void testInputINDELs() {
-        WalkerTestSpec spec = new WalkerTestSpec(
-                "-T CalculateGenotypePosteriors --no_cmdline_in_header -useACoff" +
-                        " -o %s" +
-                        " -R " + b37KGReference +
+    @Test
+    public void testInputINDELs() throws IOException {
+        IntegrationTestSpec spec = new IntegrationTestSpec(
+                        "-useACoff" +
+                        " -O %s" +
+                        " -R " + b37_reference_20_21 +    //NOTE: we need a reference for -L
                         " -L 20:10,000,000-10,100,000" +
-                        " -V " + validationDataLocation + "NA12878.Jan2013.haplotypeCaller.subset.indels.vcf" +
-                        " -supporting " + validationDataLocation + "1000G.phase3.broad.withGenotypes.chr20.1MB.vcf",
-                1,
-                Arrays.asList("cc59ceb6dab620a353edf03ef14090f1"));
-        executeTest("testInputINDELs", spec);
+                        " -V " + dir + "NA12878.Jan2013.haplotypeCaller.subset.indels.vcf" +
+                        " -supporting " + largeDir + "1000G.phase3.broad.withGenotypes.chr20.10100000.vcf",
+                Collections.singletonList(dir + "expectedCGP_testInputINDELs.vcf")
+        );
+        spec.executeTest("testInputINDELs", this);
     }
 
-    @Test(enabled = true)
-    public void testFamilyPriors() {
-        WalkerTestSpec spec = new WalkerTestSpec(
-                "-T CalculateGenotypePosteriors --no_cmdline_in_header -useACoff" +
-                        " -o %s" +
-                        " -R " + b37KGReference +
+    @Test
+    public void testFamilyPriors() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                        "-useACoff" +
+                        " -O %s" +
                         " -ped " + CEUtrioFamilyFile +
                         " -V " + CEUtrioTest +
                         " -supporting " + CEUtrioPopPriorsTest,
-                1,
-                Arrays.asList("c7d35ce5f3675528fc484baa1c5df7b4"));
-        executeTest("testFamilyPriors", spec);
+                Collections.singletonList(dir + "expectedCGP_testFamilyPriors.vcf")
+        );
+        spec.executeTest("testFamilyPriors", this);
     }
 
-    @Test(enabled = true)
-    public void testSingleParentFamily() {
-        WalkerTestSpec spec = new WalkerTestSpec(
-                "-T CalculateGenotypePosteriors --no_cmdline_in_header " +
-                        " -o %s" +
-                        " -R " + b37KGReference +
+    @Test
+    public void testSingleParentFamily() throws IOException {
```

Comment:
Nope, looks like I never got around to doing parent-offspring pairs.  PPs are omitted.